Sources/ForemWebView/ForemWebView.swift

```diff
@@ -40,42 +40,36 @@ open class ForemWebView: WKWebView {
     }()
 
     required public init?(coder: NSCoder) {
-        super.init(coder: coder)
+        let customConfig = ForemWebView.configuration()
+        super.init(frame: UIScreen.main.bounds, configuration: customConfig)
         setupWebView()
     }
 
     public override init(frame: CGRect, configuration: WKWebViewConfiguration) {
-        super.init(frame: frame, configuration: configuration)
+        let customConfig = ForemWebView.configuration(base: configuration)
+        super.init(frame: frame, configuration: customConfig)
         setupWebView()
     }
```

Comment:
I found that assigning `applicationNameForUserAgent` **after** the `WKWebView` was instantiated (after `init`) didn't work properly. 

Because of the above, a class (static) func is introduced to make this configuration possible and the rest of the setup is done after instantiation (in the `setupWebView` func).